_posts/2020-10-01-annual-report-part-3.md

```diff
@@ -0,0 +1,70 @@
+
+We also understand that active measures are more effective than passive advocacy of diversity and inclusion. Therefore, during our break between OLS-1 and OLS-2 we continued to proactively deliver workshops, public resources and seminars in order to reach out to researchers who are traditionally underrepresented in open science.
+
+We hosted 2 webinars before launching the OLS-2 call for application and presented the project report of the pilot program at the following international computation and education-related conferences (virtual conferences due to COVID-19 pandemic):
+- **[Bioinformatics Community Conference 2020](https://bcc2020.github.io/)**: short talks and [3 workshops](https://bcc2020.sched.com/event/c44n/building-communities-with-open-source-open-science) (3 hours each) across 3 time zones.
+- **Open Science Seminar at Eastern Africa Network for Bioinformatics Training (EANBiT) 2020**: Participated in a training panel (2 hours)
+- **Collaboration Workshop 2020 by Software Sustainability Institute, UK**: lightning talk and a workshop (1.5 hours)
+- **CarpentryCon @ Home 2020**: 2 workshops on [Open Leaders Fundamentals](https://2020.carpentrycon.org/schedule/#session-45) and [Ally skills](https://2020.carpentrycon.org/schedule/#session-46) (1.5 hours each)
+- **Research Data Alliance Early Career and Engagement IG 2020**: Webinar (1 hour)
+
+These engagements helped OLS become more visible across different research and open source communities. We will continue to do so to connect with those who are currently not in OLS, and Open Science at large.
+
+## Planning for the long-term sustainability
+
+Going forward, we would expand and improve our resources to develop customisable training and mentoring programs to meet the needs of individuals from different communities with varying levels of resources and institutional support available to them.
+Currently, our time in OLS is not funded and is largely after-hours, and financially we are not in a position to onboard new members who can work with us to develop these curriculum. 
+
+We have learned the hard way that non-profit projects like OLS come with financial and personal challenges. These challenges will keep us from working towards our goals until we have developed a stable funding model and business strategy to ensure the sustainability of this project.
+
+In the short term, we have planned to offer an Ally Skills training session that will be open to public registration as a way to raise some ready cash for the organisation. Attendance is free to OLS members, with varying ticket prices for individuals who are able to pay, and fee waivers available if interested members otherwise could not attend. A separate post will follow about our Ally Skills workshop to raise funding for OLS-2.
```

Comment:
```suggestion
In the short term, we are offering an Ally Skills training session that will be open to public registration as a way to raise some ready cash for the organisation. Attendance is free to OLS members, with varying ticket prices for individuals who are able to pay, and fee waivers available if interested members otherwise could not attend. A separate post will follow about our Ally Skills workshop to raise funding for OLS-2.
```